src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -228,6 +242,20 @@ public void deleteTickets(long id, long... ids) {
     public Iterable<Ticket> getRecentTickets() {
         return new PagedIterable<Ticket>(cnst("/tickets/recent.json"), handleList(Ticket.class, "tickets"));
     }
+    
+    public Iterable<Ticket> getTicketsIncrementally(Date startTime) {
+        return new PagedIterable<Ticket>(
+                tmpl("/incremental/tickets.json{?start_time}").set("start_time", (int) (startTime.getTime() / 1000)), 
+                handleIncrementalList(Ticket.class, "tickets"));                
+    }
+    
+    public Iterable<Ticket> getTicketsIncrementally(Date startTime, Date endTime) {
+        return new PagedIterable<Ticket>(
+                tmpl("/incremental/tickets.json{?start_time,end_time}")
+                    .set("start_time", (int) (startTime.getTime() / 1000))
```

Comment:
for readability I'd suggest creating a small private method which maybe takes a date divides by 1000 and casts, looks a ugly over and over.